### ISSUE TITLE
Missing important attribute structures

### ISSUE THREAD

**Issue by djaonourside** [kLabz/haxe-react-types#2]
For instance, ButtonHTMLAttributes, CanvasHTMLAttributes, InputHTMLAttributes and so on.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L1903

**Comment by kLabz** [comment]
Yeah I know, that's something I want to add someday.. PRs welcome ;)
